cmd/quadlet/main.go

```diff
@@ -183,9 +180,16 @@ func appendSubPaths(dirs []string, path string, isUserFlag bool, filterPtr func(
 }
 
 func nonNumericFilter(_path string, isUserFlag bool) bool {
+	UnitDirAdminUser := filepath.Join(quadlet.UnitDirAdmin, "users")
```

Comment:
Maybe I'm missing something here, but can't this code (and the one in `userLevelFilter`) be executed once somewhere else?

Reply:
I guess I wasn't clear on my first comment. I wonder if we can compute `UnitDirAdminUser` and `resolvedUnitDirAdmin` once instead of every time the filter function is called.